sdks/python/apache_beam/transforms/core.py

```diff
@@ -2320,6 +2359,7 @@ def expand(self, pcoll):
     return super(WindowInto, self).expand(pcoll)
 
   def to_runner_api_parameter(self, context):
+    # type: (PipelineContext) -> typing.Tuple[str, message.Message]
```

Comment:
typing.Tuple[str, Union[message.Message, bytes]]

Reply:
Looking at the code I don't see anywhere that `WindowInto.to_runner_api_parameter()` returns `bytes`.   This function actually returns `beam_runner_api_pb2.WindowingStrategy` by way of `self.windowing.to_runner_api(context)` (just below this).  I also don't see any subclasses of `WindowInto` that override this method.

Reply:
Oh, yes, this is the method on the transform, not the WindowFn. Fine as is.